contracts/Liquidator/Liquidator.sol

```diff
@@ -90,10 +111,16 @@ contract Liquidator is Ownable2StepUpgradeable, ReentrancyGuardUpgradeable, Liqu
     event NewMinLiquidatableVAI(uint256 oldMinLiquidatableVAI, uint256 newMinLiquidatableVAI);
 
     /// @notice Emitted when force liquidation is paused
-    event ForceVaiLiquidationPaused(address sender);
+    event ForceVAILiquidationPaused(address indexed sender);
 
     /// @notice Emitted when force liquidation is resumed
-    event ForceVaiLiquidationResumed(address sender);
+    event ForceVAILiquidationResumed(address indexed sender);
+
+    /// @notice Emitted when new address of protocol share reserve is set
+    event NewProtocolShareReserve(address indexed oldProtocolShareReserve, address indexed newProtocolShareReserves);
+
+    /// @notice Emitted when reserves are reduced from liquidator contract to protocol share reserves
+    event ReservesReduced(address indexed sender, address indexed token, uint256 reducedAmount);
```

Comment:
I would use a different name for this event. I assume you chose this name to make it equal to the event `ReservesReduced` emitted in the VToken contracts, but I would be explicit with the context where this is happening and I would name it something like: `ProtocolLiquidationIncentiveTransferred(address indexed liquidator, address indexed token, uint256 amount)`

Reply:
Fixed [here](https://github.com/VenusProtocol/venus-protocol/pull/259/commits/597a2ec13591bbbb92f06343fee721180e90db2c)